adi/jesd_internal.py

```diff
@@ -5,7 +5,94 @@
 from .sshfs import sshfs
 
 
-class jesd:
+class eye_scan(object):
+    _jesd_es_duration_ms = 100
+    _jesd_prbs = 7
+
+    _half_rate = {"mode": "Half Fate", "scale": 0.004}
+    _quarter_rate = {"mode": "Quarter Rate", "scale": 0.001}
+
+    lanes = {}
+
+    def get_eye_data(self, lanes=None):
```

Comment:
Not sure if this belongs into jesd_internal 
This is a MxFE side JRX eye_scan and is specific to MxFE...
I would assume this should go into ad9081.py?

Reply:
I reorganized the inheritance so this will be only pulled in by MxFE